src/styles/cylc/_dashboard.scss

```diff
@@ -0,0 +1,33 @@
+.c-dashboard {
+  padding: 0 24px;
+  /* Items in the bottom menu are left-aligned with the top Workflows&Events
+   entries, so we are required to override some styles from Vuetify components
+   in here.*/
+
+  /* to left align items in the dashboard */
+  .v-avatar .v-icon {
+    width: auto;
+    height: auto;
+  }
+  .v-list__tile__action, .v-list__tile__avatar {
+    min-width: auto;
+    margin-left: -10px;
+  }
+
+  /* otherwise avatar does not left align with component above,
+     !important as the attributes are set in the tag element -->
```

Comment:
erroneous end comment?

Reply:
I'm impressed somehow it appears to be parsed correctly? Even `vim` is interpreting the next line as non-comment.

![image](https://user-images.githubusercontent.com/304786/70485575-3a726600-1b54-11ea-8a6d-a77f870e9736.png)

Reply:
Fixed.